www/js/test.js

```diff
@@ -450,10 +450,7 @@ function UpdateSettingsSummary()
     if( conn != undefined )
         summary += ", " + conn.replace(/\((.)*\)/,'') + " connection";
         
-    if( $('#keep_test_private').attr('checked') )
-        summary += ", private";
-    else
-        summary += ", results are public";
+    summary += ", results are private";

```

Comment:
It differs based on the setting (for private instances). Probably best to just remove the private/public indicator from the summary.

Reply:
Thanks.   Just updated to remove the public/private indicator from the summary.